docs/UserGuide.adoc

```diff
@@ -195,10 +206,10 @@ Returns `john` and `John Doe`
 * `find Betsy Tim John` +
 Returns any person having names `Betsy`, `Tim`, or `John`
 
-=== Deleting a person/event : `delete`
+=== Deleting a person/event : `delete(E)`
```

Comment:
Commands shoud be easy to type :-p

Reply:
Hi prof!

Actually, we try to use the additional `E` to distinguish events/contacts commands. Basically, `E` stands for `Event`. This should be eaiser for users to remember.

Is that better if we make the commands for events/contacts to be the same? This needs a major refactor of the model component, though.

Btw, should this be called "_refactor_" or not? Refactor should be in small steps, in this sense such changes should not be called refactor anymore. Haha

Reply:
My comment is about the use of brackets.  Regular letters are faster to type than symbols and using 2 symbols seems a waste of typing. Look for an alternative syntax that is easier to type.

Reply:
Oh, actually it should be:
- delete a person: `delete`
- delete an event: `deleteE`

Users will not type the pair of brackets.

Sorry for the confusion.